client/extensions/woocommerce/package.json

```diff
@@ -1,12 +1,12 @@
 {
 	"name": "woocommerce",
 	"author": "Automattic",
-	"description": "WooCommerce",
+	"description": "WooCommerce Store",
 	"version": "0.0.1",
 	"env_id": [ "development", "wpcalypso" ],
 	"section": {
-		"name": "woocommerce",
-		"paths": [ "/woocommerce" ],
+		"name": "store",
```

Comment:
I'd keep the name as woocommerce as it will be used for the bundle name, and it's clearer there.